api/courses.go

```diff
@@ -154,10 +155,14 @@ func (r coursesRoutes) getLive(c *gin.Context) {
 				lectureHall = &lh
 			}
 		}
+
+		var stat model.Stat
+		stat, err = r.StatisticsDao.GetStat(stream.ID)
```

Comment:
We should be able to just get this number from memory. The length of the chat session contains the number of viewers: https://github.com/joschahenningsen/TUM-Live/blob/a1c9209f5a875eab2967035d742ecab0b2e56fac/api/chat.go#LL661C43-L661C43

Would this be easier to implement and more efficient?

Reply:
Definitely. 👍🏻 However, we need to make sure to update this if we ever shard and distribute the chat as in #889.